src/Kunstmaan/NodeBundle/Form/NodeMenuTabTranslationAdminType.php

```diff
@@ -30,6 +42,22 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             'attr' => ['title' => 'kuma_node.form.menu_tab_translation.weight.title'],
             'choice_translation_domain' => false,
         ]);
+
+        $builder->addEventListener(FormEvents::POST_SET_DATA, function (FormEvent $event) {
+            $nt = $event->getData();
+            if ($nt instanceof NodeTranslation && $nt->getNode()->getParent() !== null) {
+                $maxWeight = $this->em->getRepository(NodeTranslation::class)->getMaxChildrenWeight($nt->getNode()->getParent());
+
+                $event->getForm()->add('weight', ChoiceType::class, [
+                    'label' => 'kuma_node.form.menu_tab_translation.weight.label',
+                    'choices' => array_combine(range(0-$maxWeight, $maxWeight+1), range(0-$maxWeight, $maxWeight+1)),
+                    'required' => false,
+                    'placeholder' => false,
+                    'attr' => ['title' => 'kuma_node.form.menu_tab_translation.weight.title'],
+                    'choice_translation_domain' => false,
```

Comment:
@Numkil Check if you can refactor this to get the original options and only change the choices. To avoid duplicating all options

eg.
```php
$options = $form->get('weight')->getConfig()->getOptions();
// Change options
$form->add('weight', ChoiceType::class, $options);
```